README.md

```diff
@@ -107,7 +122,7 @@ You can also completely omit the output of a line-height declaration:
 ```scss
 .no-line-height {
     @include fluidms-font-size(
-        $ms: 3,
+        $font-size: ms(3),
         $line-height: false
```

Comment:
Document what the `false` value results in

Reply:
I tried to clarify more how to disabled line-height output in 16a5e13